Code/Registration/include/sitkImageRegistrationMethod.h

```diff
@@ -495,6 +495,15 @@ namespace simple
     /** \brief Get the percentage of pixels used for metric evaluation.  */
     const std::vector<double> &GetMetricSamplingPercentagePerLevel() const;
 
+    /** \brief Sampling strategies for obtaining points.
+     *
+     * - NONE: use all voxels, sampled points are the voxel centers.
+     * - REGULAR: sample image voxels using a regular grid, then within
+     *            each voxel randomly perturb from center.
+     * - RANDOM: sample image voxels with replacement using a uniform
```

Comment:
what does with replacement mean?  probably just me being dumb.

Reply:
The phrase is using the statistics nomenclature, we are performing a simple random sampling with replacement. Uniform probability for selecting each voxel and we don't remove the voxel once it is selected so "with replacement". This means the same voxel can be sampled multiple times (very small probability, but still greater than zero). If we used sampling without replacement the probability of selecting the same voxel multiple times is zero.